src/client/Microsoft.Identity.Client/ManagedIdentity/ManagedIdentitySourceType.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.ManagedIdentity
+{
+    /// <summary>
+    /// Managed identity sources supported by MSAL.
+    /// </summary>
+    public enum ManagedIdentitySourceType
```

Comment:
Naming `ManagedIdentitySource` ?

CC @localden

Reply:
Updated the enum to ManagedIdentitySource.